src/base-element.js

```diff
@@ -76,6 +76,11 @@ import {vsyncFor} from './vsync';
  *           \/
  *    State: <IN VIEWPORT>
  *
+ * The preconnectCallback is called when the systems thinks it is good
```

Comment:
I didn't on purpose.

It isn't part of the state machine. I could put "might be called after this" (as I wrote in the text), but we may call it 100 times at random times and your program should not break.